ofrak_core/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - View resource attribute string values containing only digits primarily as strings, alternatively as hex numbers. ([#423](https://github.com/redballoonsecurity/ofrak/pull/423))
 
 ### Changed
+- Refactored frontend to do away with the dataLenPromise.then() calls and store the dataLength and resourceNodeDataMap in a store. ([#434](https://github.com/redballoonsecurity/ofrak/pull/434))
```

Comment:
@whyitfor may want to weigh in on this, but since it's an internal refactor of some code, I'm not sure if we even need a CHANGELOG entry for this.